infra/event-sync/src/main/java/com/simprints/infra/eventsync/sync/up/tasks/EventUpSyncTask.kt

```diff
@@ -143,7 +143,7 @@ internal class EventUpSyncTask @Inject constructor(
                 throw t
             }
 
-            Simber.e("Failed to upload event scopes", t)
+            Simber.tag(SYNC).e("Failed to upload event scopes", t)
```

Comment:
Wouldn't this be reported by the worker as well?

Reply:
In this case "t" is not propagated out of the catch block, therefore it would not be logged in worker.